db/migrations/postgres/000033_create_sidebar_categories.down.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE sidebarcategories DROP COLUMN IF EXISTS collapsed;
+ALTER TABLE sidebarcategories DROP COLUMN IF EXISTS muted
+
+DO $$
+<<modify_column_type_if_type_is_different>>
+DECLARE
+    col_exist_and_type_different boolean := false;
+BEGIN
+    SELECT count(*) != 0 INTO col_exist_and_type_different
+    FROM information_schema.columns
+    WHERE table_name = 'sidebarcategories'
+    AND column_name = 'id'
+    AND data_type = 'character varying'
+    AND NOT character_maximum_length = 26;
```

Comment:
Is this any different than using `!=` or `<>` instead of the equality check?

Reply:
Hmm, probably not but it looks more like sql-ish :p